### DISCUSSION TITLE
AttributeError: module 'pysr' has no attribute 'Problem'

### DISCUSSION THREAD

**Discussion by wkharold** [MilesCranmer/PySR#665] [Q&A] [answered]
I'm just getting started with PySR. Walking through the Toy Examples with Code. When I do 
```python
from pysr import *
```
I get the error in the title:
```
AttributeError: module 'pysr' has no attribute 'Problem'
```
I am running IPython in an Apptainer container built from the `continuumio/miniconda3` Docker image.  

When I switch to 
```python
import pysr
```
and preface things with `pysr.` as required everything works as expected.

**Comment by MilesCranmer** [comment, marked as answer]
Looks like a bug. I made an issue #666 to track it.

Edit: Fixed.
